libdav1d 0.9.2 on jammy isn't new enough for Chromium's taste:

  third_party/webrtc/modules/video_coding/codecs/av1/dav1d_decoder.cc:87:5: error: no member named 'n_threads' in 'Dav1dSettings'
    s.n_threads = std::max(2, settings.number_of_cores());
    ~ ^
  third_party/webrtc/modules/video_coding/codecs/av1/dav1d_decoder.cc:88:5: error: no member named 'max_frame_delay' in 'Dav1dSettings'
    s.max_frame_delay = 1;   // For low latency decoding.
    ~ ^

  media/filters/dav1d_video_decoder.cc:201:5: error: no member named 'n_threads' in 'Dav1dSettings'
    s.n_threads = VideoDecoder::GetRecommendedThreadCount(frame_threads *
    ~ ^
  media/filters/dav1d_video_decoder.cc:207:7: error: no member named 'max_frame_delay' in 'Dav1dSettings'
    s.max_frame_delay = 1;
    ~ ^

  error[E0609]: no field `max_frame_delay` on type `dav1d_sys::bindings::Dav1dSettings`
    --> ../../third_party/crabbyavif/src/src/codecs/dav1d.rs:40:18
     |
  40 |         settings.max_frame_delay = 1;
     |                  ^^^^^^^^^^^^^^^ unknown field

  error[E0609]: no field `n_threads` on type `dav1d_sys::bindings::Dav1dSettings`
    --> ../../third_party/crabbyavif/src/src/codecs/dav1d.rs:41:18
     |
  41 |         settings.n_threads = 8;
     |                  ^^^^^^^^^ unknown field

Fortunately, these are easy to work around.

--- a/media/filters/dav1d_video_decoder.cc
+++ b/media/filters/dav1d_video_decoder.cc
@@ -296,6 +296,7 @@ void Dav1dVideoDecoder::Initialize(const
   GetDecoderThreadCounts(config.coded_size().height(), &tile_threads,
                          &frame_threads);
 
+#if 0	// XtraDeb fix
   // While dav1d has switched to a thread pool, preserve the same thread counts
   // we used when tile and frame threads were configured distinctly. It may be
   // possible to lower this after some performance analysis of the new system.
@@ -307,6 +308,7 @@ void Dav1dVideoDecoder::Initialize(const
   if (low_delay) {
     s.max_frame_delay = 1;
   }
+#endif	// XtraDeb fix
 
   // Only output the highest spatial layer.
   s.all_layers = 0;
--- a/third_party/crabbyavif/src/src/codecs/dav1d.rs
+++ b/third_party/crabbyavif/src/src/codecs/dav1d.rs
@@ -143,12 +143,14 @@ impl Dav1d {
         unsafe { dav1d_default_settings(settings_uninit.as_mut_ptr()) };
         // # Safety: settings_uninit was initialized in the C function above.
         let mut settings = unsafe { settings_uninit.assume_init() };
+/* XtraDeb fix
         if low_latency {
             settings.max_frame_delay = 1;
         }
         settings.n_threads = i32::try_from(config.max_threads)
             .unwrap_or(1)
             .clamp(1, DAV1D_MAX_THREADS as _);
+*/
         settings.operating_point = config.operating_point as i32;
         settings.all_layers = if config.all_layers { 1 } else { 0 };
         let frame_size_limit = match config.image_size_limit {
--- a/third_party/webrtc/modules/video_coding/codecs/av1/dav1d_decoder.cc
+++ b/third_party/webrtc/modules/video_coding/codecs/av1/dav1d_decoder.cc
@@ -98,8 +98,10 @@ bool Dav1dDecoder::Configure(const Setti
   Dav1dSettings s;
   dav1d_default_settings(&s);
 
+#if 0	// XtraDeb fix
   s.n_threads = std::clamp(settings.number_of_cores(), 1, DAV1D_MAX_THREADS);
   s.max_frame_delay = 1;  // For low latency decoding.
+#endif
   s.all_layers = 0;       // Don't output a frame for every spatial layer.
   // Limit max frame size to avoid OOM'ing fuzzers. crbug.com/325284120.
   s.frame_size_limit = 16384 * 16384;
